src/xtgeo/cube/cube1.py

```diff
@@ -948,7 +949,14 @@ def from_roxar(self, project, name, folder=None):  # pragma: no cover
         self._metadata.required = self
 
     def to_roxar(
-        self, project, name, folder=None, domain="time", compression=("wavelet", 5)
+        self,
+        project: Any,
+        name: str,
+        folder: Optional[str] = None,
+        propname: str = "seismic_attribute",
+        domain: str = "time",
+        compression: tuple = ("wavelet", 5),
```

Comment:
Maybe be more precise with `Tuple[str, int]`? Can this also take a float? Maybe the docstring could also say this?